package.json

```diff
@@ -1,24 +0,0 @@
-{
-  "name": "leetcode",
-  "version": "1.0.0",
-  "main": "index.js",
-  "dependencies": {
-    "eslint": "^7.27.0",
-    "jest": "^27.0.2"
-  },
-  "devDependencies": {},
```

Comment:
These two are semantically `devDependencies`.

Reply:
Well spotted, thank you. Pretty sure I uninstalled them and then installed them again with -D, but I had some troubles with mixed master and main branches, so I must have pulled the old package.json file when I deleted the master branch.

Reply:
👍 
The `main` branch, not `master` - we're being inclusive, aren't we? :)

Reply:
Ah, ok, sorry, I reread it and now know what you mean. :) 
👍